tools/ZAPD/ZAPD/ZTexture.cpp

```diff
@@ -835,13 +835,13 @@ TextureType ZTexture::GetTextureTypeFromString(std::string str)
 		texType = TextureType::RGBA32bpp;
 	else if (str == "rgba16")
 		texType = TextureType::RGBA16bpp;
-	else if (str == "rgb5a1")
+	else if (str == "rgba16")
```

Comment:
```suggestion
	else if (str == "rgb5a1")
```
monkaS

Reply:
Somebody forgot to exclude `tools/` from the global find and replace D:

Reply:
fig is sabotaging zapd, you heard it here first